pkg/distributor/ha_tracker_test.go

```diff
@@ -70,6 +70,53 @@ func checkReplicaTimestamp(t *testing.T, duration time.Duration, c *haTracker, u
 	})
 }
 
+func TestHATrackerCacheSyncOnStart(t *testing.T) {
+	cluster := "c1"
+	replica := "r1"
```

Comment:
nitpick to make these `const`s. Some editor color variables and constants differently for example

Reply:
Fixed [2e808ec](https://github.com/grafana/mimir/pull/9826/commits/2e808ec33b59399794bf624b2c16ea04fb8b0cf5). Thank you